gem/management/commands/migrate_surveys_to_forms.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from molo.surveys.models import (
+    MoloSurveyPage, SurveyIndexPage, MoloSurveyFormField,
+    MoloSurveyView, MoloSurveyPageSubmission)
+from molo.forms.models import (
+    MoloFormPage, FormIndexPage, MoloFormView,
+    MoloFormsFormField, MoloFormPageSubmission)
+from molo.core.models import Main
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        for main in Main.objects.all():
+            surveys_index = SurveyIndexPage.objects.child_of(main).first()
+            forms_index = FormIndexPage.objects.child_of(main).first()
```

Comment:
Don't we have these indexes for all the sites? So we'll want to change it for more than just 'Main' right?
and are there different index pages for different languages or do they use the same one?